parser/src/python.lalrpop

```diff
@@ -817,7 +795,7 @@ Atom: ast::Expression = {
         // List comprehension:
         e
     },
-    "(" <e:TestList?> <trailing_comma:","?> ")" => {
+    "(" <e:TestList2?> <trailing_comma:","?> ")" => {
```

Comment:
Why do we need this part in the first place?
`TestList` already parses `1, 2, 3` as a tuple, and wrapping it in `()` should be a no-op, like any other expression.

This seems to work for me (passes all tests, didn't look for edge cases):
```
    "(" ")" => ast::Expression::Tuple { elements: vec![] },
    "(" <e:TestList> ")" => { e },
```
And then (and after resolving the issue above) we can drop `TestList2`.

Reply:
I resolved this, and dropped TestList2. Thank you for the review!